docs/dev/kubernetes/kubernetes_basic_workflow.md

```diff
@@ -0,0 +1,29 @@
+# Kubernetes Basic Transformation Workflow
+
+This document outlines the workflow for the deployment of a simple application on the Kubernetes platform in order to help design the basic Kubernetes plugin for TOSCAna.
+
+#### Basic Information about Kubernetes
+- Packs applications in isolated containers instead of VMs (one application per container)
+- Deploys containers based on operating-system-level virtualization rather than hardware virtualization
+- Kubernetes mainly uses Docker for containers
+
+## Automated Workflow
+Describes the planned automated deployment of the simple application based on our [transformation workflow](https://github.com/StuPro-TOSCAna/TOSCAna/blob/transformation-flow/docs/dev/architecture/workflow.md).
+
+### Transformation
+- Data model of the simple app is provided by core
+- Application is already containerized through Docker
```

Comment:
Does that mean it's already in a registry?

Reply:
From my understanding, the application is a TODO app inside of a Docker container, so I assume it wouldnt be hard to get the container image. I commented below regarding the registry issue.